pkgs/development/python-modules/django-mptt/default.nix

```diff
@@ -2,35 +2,34 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  hatchling,
   django,
   django-js-asset,
-  python,
 }:
 
 buildPythonPackage rec {
   pname = "django-mptt";
-  version = "0.13.4";
-  format = "setuptools";
+  version = "0.16";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
```

Comment:
```suggestion
    owner = "django-mptt";
    repo = "django-mptt";
```

Reply:
Can I ask why this change is needed? I don't understand the need to hardcode this. Trying to get the hang of "why we do things how we do things" 😄